notification-service-sdk/pom.xml

```diff
@@ -174,11 +174,26 @@
             <groupId>org.mockito</groupId>
             <artifactId>mockito-core</artifactId>
         </dependency>
+        <dependency>
+            <groupId>javax</groupId>
+            <artifactId>javaee-web-api</artifactId>
+            <version>6.0</version>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>jakarta.ws.rs</groupId>
+            <artifactId>jakarta.ws.rs-api</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-api</artifactId>
+            <scope>test</scope>
+        </dependency>
     </dependencies>
     <properties>
         <gson-fire-version>1.8.5</gson-fire-version>
         <swagger-core-version>1.6.3</swagger-core-version>
         <jackson-databind-nullable-version>0.2.2</jackson-databind-nullable-version>
-        <openapi.generator.version>5.4.0</openapi.generator.version>
+        <openapi.generator.version>6.2.0</openapi.generator.version>
```

Comment:
please make it consistent with workflow-service-sdk

Reply:
Moved this change to different PR
Please see PR #289